bundler/onProdDone.js

```diff
@@ -0,0 +1,2 @@
+function onProdDone (stats) {}
```

Comment:
...yeah... so it is like a "hook" file, 

inside you should write any methods you want to be triggered after 'done' event on prod build:
https://github.com/snipsco/teleport-webpack-react/blob/upgrade-webpack2/bundler/prod.config.js#L63

(because in teleport logic, you want to build other templates that will be still using this prod.config.js file here (and not override it))

for example, I am building a template that helps to use automatically a react-router redux framework with server side rendering:
https://github.com/Ledoux/teleport-react-router/blob/master/bundler/onProdDone.js#L4

in this template I use onProdDone to build all my server side rendered assets html file.